### ISSUE TITLE
Cannot install protobuf PHP 7.4.10 on OSX 10.15.7

### ISSUE THREAD

**Issue by Plytas** [protocolbuffers/protobuf#7918]
**What version of protobuf and what language are you using?**
Version: 3.13.0
Language: PHP 7.4.10
OS: macOS Catalina 10.15.7

**What did you do?**
1. `pecl install protobuf`
2. See error

**What did you expect to see**
I expected protobuf to install correctly

**What did you see instead?**
protobuf did not install.

Output:
```BASH
$ pecl install protobuf
downloading protobuf-3.13.0.tgz ...
Starting to download protobuf-3.13.0.tgz (192,753 bytes)
.........................................done: 192,753 bytes
21 source files, building
running: phpize
Configuring for:
PHP Api Version:         20190902
Zend Module Api No:      20190902
Zend Extension Api No:   320190902
building in /private/tmp/pear/temp/pear-build-vytasCIXA8R/protobuf-3.13.0
running: /private/tmp/pear/temp/protobuf/configure --with-php-config=/usr/local/opt/php/bin/php-config
checking for grep that handles long lines and -e... /usr/bin/grep
checking for egrep... /usr/bin/grep -E
checking for a sed that does not truncate output... /usr/bin/sed
checking for pkg-config... /usr/local/bin/pkg-config
checking pkg-config is at least version 0.9.0... yes
checking for cc... cc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables...
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether cc accepts -g... yes
checking for cc option to accept ISO C89... none needed
checking how to run the C preprocessor... cc -E
checking for icc... no
checking for suncc... no
checking for system library directory... lib
checking if compiler supports -R... no
checking if compiler supports -Wl,-rpath,... yes
checking build system type... x86_64-apple-darwin19.6.0
checking host system type... x86_64-apple-darwin19.6.0
checking target system type... x86_64-apple-darwin19.6.0
checking for PHP prefix... /usr/local/Cellar/php/7.4.10
checking for PHP includes... -I/usr/local/Cellar/php/7.4.10/include/php -I/usr/local/Cellar/php/7.4.10/include/php/main -I/usr/local/Cellar/php/7.4.10/include/php/TSRM -I/usr/local/Cellar/php/7.4.10/include/php/Zend -I/usr/local/Cellar/php/7.4.10/include/php/ext -I/usr/local/Cellar/php/7.4.10/include/php/ext/date/lib
checking for PHP extension directory... /usr/local/Cellar/php/7.4.10/pecl/20190902
checking for PHP installed headers prefix... /usr/local/Cellar/php/7.4.10/include/php
checking if debug is enabled... no
checking if zts is enabled... no
checking for gawk... no
checking for nawk... no
checking for awk... awk
checking if awk is broken... no
checking whether to enable Protobuf extension... yes, shared
checking for a sed that does not truncate output... /usr/bin/sed
checking for ld used by cc... /Library/Developer/CommandLineTools/usr/bin/ld
checking if the linker (/Library/Developer/CommandLineTools/usr/bin/ld) is GNU ld... no
checking for /Library/Developer/CommandLineTools/usr/bin/ld option to reload object files... -r
checking for BSD-compatible nm... /usr/bin/nm -B
checking whether ln -s works... yes
checking how to recognize dependent libraries... pass_all
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking dlfcn.h usability... yes
checking dlfcn.h presence... yes
checking for dlfcn.h... yes
checking the maximum length of command line arguments... 196608
checking command to parse /usr/bin/nm -B output from cc object... ok
checking for objdir... .libs
checking for ar... ar
checking for ranlib... ranlib
checking for strip... strip
checking for dsymutil... dsymutil
checking for nmedit... nmedit
checking for -single_module linker flag... yes
checking for -exported_symbols_list linker flag... yes
checking if cc supports -fno-rtti -fno-exceptions... yes
checking for cc option to produce PIC... -fno-common
checking if cc PIC flag -fno-common works... yes
checking if cc static flag -static works... no
checking if cc supports -c -o file.o... yes
checking whether the cc linker (/Library/Developer/CommandLineTools/usr/bin/ld) supports shared libraries... yes
checking dynamic linker characteristics... darwin19.6.0 dyld
checking how to hardcode library paths into programs... immediate
checking whether stripping libraries is possible... yes
checking if libtool supports shared libraries... yes
checking whether to build shared libraries... yes
checking whether to build static libraries... no

creating libtool
appending configuration tag "CXX" to libtool
configure: patching config.h.in
configure: creating ./config.status
config.status: creating config.h
running: make
/bin/sh /private/tmp/pear/temp/pear-build-vytasCIXA8R/protobuf-3.13.0/libtool --mode=compile cc  -I. -I/private/tmp/pear/temp/protobuf -DPHP_ATOM_INC -I/private/tmp/pear/temp/pear-build-vytasCIXA8R/protobuf-3.13.0/include -I/private/tmp/pear/temp/pear-build-vytasCIXA8R/protobuf-3.13.0/main -I/private/tmp/pear/temp/protobuf -I/usr/local/Cellar/php/7.4.10/include/php -I/usr/local/Cellar/php/7.4.10/include/php/main -I/usr/local/Cellar/php/7.4.10/include/php/TSRM -I/usr/local/Cellar/php/7.4.10/include/php/Zend -I/usr/local/Cellar/php/7.4.10/include/php/ext -I/usr/local/Cellar/php/7.4.10/include/php/ext/date/lib  -DHAVE_CONFIG_H  -g -O2   -c /private/tmp/pear/temp/protobuf/arena.c -o arena.lo
mkdir .libs
 cc -I. -I/private/tmp/pear/temp/protobuf -DPHP_ATOM_INC -I/private/tmp/pear/temp/pear-build-vytasCIXA8R/protobuf-3.13.0/include -I/private/tmp/pear/temp/pear-build-vytasCIXA8R/protobuf-3.13.0/main -I/private/tmp/pear/temp/protobuf -I/usr/local/Cellar/php/7.4.10/include/php -I/usr/local/Cellar/php/7.4.10/include/php/main -I/usr/local/Cellar/php/7.4.10/include/php/TSRM -I/usr/local/Cellar/php/7.4.10/include/php/Zend -I/usr/local/Cellar/php/7.4.10/include/php/ext -I/usr/local/Cellar/php/7.4.10/include/php/ext/date/lib -DHAVE_CONFIG_H -g -O2 -c /private/tmp/pear/temp/protobuf/arena.c  -fno-common -DPIC -o .libs/arena.o
/bin/sh /private/tmp/pear/temp/pear-build-vytasCIXA8R/protobuf-3.13.0/libtool --mode=compile cc  -I. -I/private/tmp/pear/temp/protobuf -DPHP_ATOM_INC -I/private/tmp/pear/temp/pear-build-vytasCIXA8R/protobuf-3.13.0/include -I/private/tmp/pear/temp/pear-build-vytasCIXA8R/protobuf-3.13.0/main -I/private/tmp/pear/temp/protobuf -I/usr/local/Cellar/php/7.4.10/include/php -I/usr/local/Cellar/php/7.4.10/include/php/main -I/usr/local/Cellar/php/7.4.10/include/php/TSRM -I/usr/local/Cellar/php/7.4.10/include/php/Zend -I/usr/local/Cellar/php/7.4.10/include/php/ext -I/usr/local/Cellar/php/7.4.10/include/php/ext/date/lib  -DHAVE_CONFIG_H  -g -O2   -c /private/tmp/pear/temp/protobuf/array.c -o array.lo
 cc -I. -I/private/tmp/pear/temp/protobuf -DPHP_ATOM_INC -I/private/tmp/pear/temp/pear-build-vytasCIXA8R/protobuf-3.13.0/include -I/private/tmp/pear/temp/pear-build-vytasCIXA8R/protobuf-3.13.0/main -I/private/tmp/pear/temp/protobuf -I/usr/local/Cellar/php/7.4.10/include/php -I/usr/local/Cellar/php/7.4.10/include/php/main -I/usr/local/Cellar/php/7.4.10/include/php/TSRM -I/usr/local/Cellar/php/7.4.10/include/php/Zend -I/usr/local/Cellar/php/7.4.10/include/php/ext -I/usr/local/Cellar/php/7.4.10/include/php/ext/date/lib -DHAVE_CONFIG_H -g -O2 -c /private/tmp/pear/temp/protobuf/array.c  -fno-common -DPIC -o .libs/array.o
/private/tmp/pear/temp/protobuf/array.c:277:65: warning: format specifies type 'long' but the argument has type 'zend_long' (aka 'long long') [-Wformat]
    zend_error(E_USER_ERROR, "Element at %ld doesn't exist.\n", index);
                                         ~~~                    ^~~~~
                                         %lld
/private/tmp/pear/temp/protobuf/array.c:323:71: warning: format specifies type 'long' but the argument has type 'int64_t' (aka 'long long') [-Wformat]
    zend_error(E_USER_ERROR, "Element at index %ld doesn't exist.\n", index);
                                               ~~~                    ^~~~~
                                               %lld
/private/tmp/pear/temp/protobuf/array.c:354:22: warning: format specifies type 'long' but the argument has type 'zend_long' (aka 'long long') [-Wformat]
                     index);
                     ^~~~~
/private/tmp/pear/temp/protobuf/array.c:522:65: warning: format specifies type 'long' but the argument has type 'zend_long' (aka 'long long') [-Wformat]
    zend_error(E_USER_ERROR, "Element at %ld doesn't exist.\n", index);
                                         ~~~                    ^~~~~
                                         %lld
4 warnings generated.
/bin/sh /private/tmp/pear/temp/pear-build-vytasCIXA8R/protobuf-3.13.0/libtool --mode=compile cc  -I. -I/private/tmp/pear/temp/protobuf -DPHP_ATOM_INC -I/private/tmp/pear/temp/pear-build-vytasCIXA8R/protobuf-3.13.0/include -I/private/tmp/pear/temp/pear-build-vytasCIXA8R/protobuf-3.13.0/main -I/private/tmp/pear/temp/protobuf -I/usr/local/Cellar/php/7.4.10/include/php -I/usr/local/Cellar/php/7.4.10/include/php/main -I/usr/local/Cellar/php/7.4.10/include/php/TSRM -I/usr/local/Cellar/php/7.4.10/include/php/Zend -I/usr/local/Cellar/php/7.4.10/include/php/ext -I/usr/local/Cellar/php/7.4.10/include/php/ext/date/lib  -DHAVE_CONFIG_H  -g -O2   -c /private/tmp/pear/temp/protobuf/bundled_php.c -o bundled_php.lo
 cc -I. -I/private/tmp/pear/temp/protobuf -DPHP_ATOM_INC -I/private/tmp/pear/temp/pear-build-vytasCIXA8R/protobuf-3.13.0/include -I/private/tmp/pear/temp/pear-build-vytasCIXA8R/protobuf-3.13.0/main -I/private/tmp/pear/temp/protobuf -I/usr/local/Cellar/php/7.4.10/include/php -I/usr/local/Cellar/php/7.4.10/include/php/main -I/usr/local/Cellar/php/7.4.10/include/php/TSRM -I/usr/local/Cellar/php/7.4.10/include/php/Zend -I/usr/local/Cellar/php/7.4.10/include/php/ext -I/usr/local/Cellar/php/7.4.10/include/php/ext/date/lib -DHAVE_CONFIG_H -g -O2 -c /private/tmp/pear/temp/protobuf/bundled_php.c  -fno-common -DPIC -o .libs/bundled_php.o
/bin/sh /private/tmp/pear/temp/pear-build-vytasCIXA8R/protobuf-3.13.0/libtool --mode=compile cc  -I. -I/private/tmp/pear/temp/protobuf -DPHP_ATOM_INC -I/private/tmp/pear/temp/pear-build-vytasCIXA8R/protobuf-3.13.0/include -I/private/tmp/pear/temp/pear-build-vytasCIXA8R/protobuf-3.13.0/main -I/private/tmp/pear/temp/protobuf -I/usr/local/Cellar/php/7.4.10/include/php -I/usr/local/Cellar/php/7.4.10/include/php/main -I/usr/local/Cellar/php/7.4.10/include/php/TSRM -I/usr/local/Cellar/php/7.4.10/include/php/Zend -I/usr/local/Cellar/php/7.4.10/include/php/ext -I/usr/local/Cellar/php/7.4.10/include/php/ext/date/lib  -DHAVE_CONFIG_H  -g -O2   -c /private/tmp/pear/temp/protobuf/convert.c -o convert.lo
 cc -I. -I/private/tmp/pear/temp/protobuf -DPHP_ATOM_INC -I/private/tmp/pear/temp/pear-build-vytasCIXA8R/protobuf-3.13.0/include -I/private/tmp/pear/temp/pear-build-vytasCIXA8R/protobuf-3.13.0/main -I/private/tmp/pear/temp/protobuf -I/usr/local/Cellar/php/7.4.10/include/php -I/usr/local/Cellar/php/7.4.10/include/php/main -I/usr/local/Cellar/php/7.4.10/include/php/TSRM -I/usr/local/Cellar/php/7.4.10/include/php/Zend -I/usr/local/Cellar/php/7.4.10/include/php/ext -I/usr/local/Cellar/php/7.4.10/include/php/ext/date/lib -DHAVE_CONFIG_H -g -O2 -c /private/tmp/pear/temp/protobuf/convert.c  -fno-common -DPIC -o .libs/convert.o
/bin/sh /private/tmp/pear/temp/pear-build-vytasCIXA8R/protobuf-3.13.0/libtool --mode=compile cc  -I. -I/private/tmp/pear/temp/protobuf -DPHP_ATOM_INC -I/private/tmp/pear/temp/pear-build-vytasCIXA8R/protobuf-3.13.0/include -I/private/tmp/pear/temp/pear-build-vytasCIXA8R/protobuf-3.13.0/main -I/private/tmp/pear/temp/protobuf -I/usr/local/Cellar/php/7.4.10/include/php -I/usr/local/Cellar/php/7.4.10/include/php/main -I/usr/local/Cellar/php/7.4.10/include/php/TSRM -I/usr/local/Cellar/php/7.4.10/include/php/Zend -I/usr/local/Cellar/php/7.4.10/include/php/ext -I/usr/local/Cellar/php/7.4.10/include/php/ext/date/lib  -DHAVE_CONFIG_H  -g -O2   -c /private/tmp/pear/temp/protobuf/def.c -o def.lo
 cc -I. -I/private/tmp/pear/temp/protobuf -DPHP_ATOM_INC -I/private/tmp/pear/temp/pear-build-vytasCIXA8R/protobuf-3.13.0/include -I/private/tmp/pear/temp/pear-build-vytasCIXA8R/protobuf-3.13.0/main -I/private/tmp/pear/temp/protobuf -I/usr/local/Cellar/php/7.4.10/include/php -I/usr/local/Cellar/php/7.4.10/include/php/main -I/usr/local/Cellar/php/7.4.10/include/php/TSRM -I/usr/local/Cellar/php/7.4.10/include/php/Zend -I/usr/local/Cellar/php/7.4.10/include/php/ext -I/usr/local/Cellar/php/7.4.10/include/php/ext/date/lib -DHAVE_CONFIG_H -g -O2 -c /private/tmp/pear/temp/protobuf/def.c  -fno-common -DPIC -o .libs/def.o
/private/tmp/pear/temp/protobuf/def.c:190:62: warning: format specifies type 'long' but the argument has type 'zend_long' (aka 'long long') [-Wformat]
    zend_error(E_USER_ERROR, "Cannot get element at %ld.\n", index);
                                                    ~~~      ^~~~~
                                                    %lld
/private/tmp/pear/temp/protobuf/def.c:293:62: warning: format specifies type 'long' but the argument has type 'zend_long' (aka 'long long') [-Wformat]
    zend_error(E_USER_ERROR, "Cannot get element at %ld.\n", index);
                                                    ~~~      ^~~~~
                                                    %lld
/private/tmp/pear/temp/protobuf/def.c:622:62: warning: format specifies type 'long' but the argument has type 'zend_long' (aka 'long long') [-Wformat]
    zend_error(E_USER_ERROR, "Cannot get element at %ld.\n", index);
                                                    ~~~      ^~~~~
                                                    %lld
/private/tmp/pear/temp/protobuf/def.c:665:62: warning: format specifies type 'long' but the argument has type 'zend_long' (aka 'long long') [-Wformat]
    zend_error(E_USER_ERROR, "Cannot get element at %ld.\n", index);
                                                    ~~~      ^~~~~
                                                    %lld
4 warnings generated.
/bin/sh /private/tmp/pear/temp/pear-build-vytasCIXA8R/protobuf-3.13.0/libtool --mode=compile cc  -I. -I/private/tmp/pear/temp/protobuf -DPHP_ATOM_INC -I/private/tmp/pear/temp/pear-build-vytasCIXA8R/protobuf-3.13.0/include -I/private/tmp/pear/temp/pear-build-vytasCIXA8R/protobuf-3.13.0/main -I/private/tmp/pear/temp/protobuf -I/usr/local/Cellar/php/7.4.10/include/php -I/usr/local/Cellar/php/7.4.10/include/php/main -I/usr/local/Cellar/php/7.4.10/include/php/TSRM -I/usr/local/Cellar/php/7.4.10/include/php/Zend -I/usr/local/Cellar/php/7.4.10/include/php/ext -I/usr/local/Cellar/php/7.4.10/include/php/ext/date/lib  -DHAVE_CONFIG_H  -g -O2   -c /private/tmp/pear/temp/protobuf/map.c -o map.lo
 cc -I. -I/private/tmp/pear/temp/protobuf -DPHP_ATOM_INC -I/private/tmp/pear/temp/pear-build-vytasCIXA8R/protobuf-3.13.0/include -I/private/tmp/pear/temp/pear-build-vytasCIXA8R/protobuf-3.13.0/main -I/private/tmp/pear/temp/protobuf -I/usr/local/Cellar/php/7.4.10/include/php -I/usr/local/Cellar/php/7.4.10/include/php/main -I/usr/local/Cellar/php/7.4.10/include/php/TSRM -I/usr/local/Cellar/php/7.4.10/include/php/Zend -I/usr/local/Cellar/php/7.4.10/include/php/ext -I/usr/local/Cellar/php/7.4.10/include/php/ext/date/lib -DHAVE_CONFIG_H -g -O2 -c /private/tmp/pear/temp/protobuf/map.c  -fno-common -DPIC -o .libs/map.o
/bin/sh /private/tmp/pear/temp/pear-build-vytasCIXA8R/protobuf-3.13.0/libtool --mode=compile cc  -I. -I/private/tmp/pear/temp/protobuf -DPHP_ATOM_INC -I/private/tmp/pear/temp/pear-build-vytasCIXA8R/protobuf-3.13.0/include -I/private/tmp/pear/temp/pear-build-vytasCIXA8R/protobuf-3.13.0/main -I/private/tmp/pear/temp/protobuf -I/usr/local/Cellar/php/7.4.10/include/php -I/usr/local/Cellar/php/7.4.10/include/php/main -I/usr/local/Cellar/php/7.4.10/include/php/TSRM -I/usr/local/Cellar/php/7.4.10/include/php/Zend -I/usr/local/Cellar/php/7.4.10/include/php/ext -I/usr/local/Cellar/php/7.4.10/include/php/ext/date/lib  -DHAVE_CONFIG_H  -g -O2   -c /private/tmp/pear/temp/protobuf/message.c -o message.lo
 cc -I. -I/private/tmp/pear/temp/protobuf -DPHP_ATOM_INC -I/private/tmp/pear/temp/pear-build-vytasCIXA8R/protobuf-3.13.0/include -I/private/tmp/pear/temp/pear-build-vytasCIXA8R/protobuf-3.13.0/main -I/private/tmp/pear/temp/protobuf -I/usr/local/Cellar/php/7.4.10/include/php -I/usr/local/Cellar/php/7.4.10/include/php/main -I/usr/local/Cellar/php/7.4.10/include/php/TSRM -I/usr/local/Cellar/php/7.4.10/include/php/Zend -I/usr/local/Cellar/php/7.4.10/include/php/ext -I/usr/local/Cellar/php/7.4.10/include/php/ext/date/lib -DHAVE_CONFIG_H -g -O2 -c /private/tmp/pear/temp/protobuf/message.c  -fno-common -DPIC -o .libs/message.o
/private/tmp/pear/temp/protobuf/message.c:269:3: error: non-void function 'Message_write_property' should return a value [-Wreturn-type]
  return;
  ^
/private/tmp/pear/temp/protobuf/message.c:276:3: error: non-void function 'Message_write_property' should return a value [-Wreturn-type]
  return;
  ^
2 errors generated.
make: *** [message.lo] Error 1
ERROR: `make' failed
```

**Comment by egorperesada** [comment]
have the same issue on Macos Catalina 10.15.6

**Comment by Plytas** [comment]
I can get `protobuf-3.12.0` to install. However, I'm trying to read a stream from Google Ads API and get a 502. Nginx logs only state:
```
upstream prematurely closed connection while reading response header from upstream
```

Not sure if it's related.

**Comment by Plytas** [comment]
Issue is now fixed with v3.13.0.1
